packages/suite/src/components/suite/DeviceMenu/index.tsx

```diff
@@ -255,7 +261,7 @@ const DeviceMenu = ({
                 }
             />
             {isOpen && (
-                <Menu>
+                <Menu id="device-menu">
```

Comment:
using refs would be a bit more idiomatic approach
https://reactjs.org/docs/refs-and-the-dom.html

Reply:
ok. while fixing this, I found some other problems, so check it out please.